src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -103,7 +104,13 @@ const AccordionSection = ({
       </div>
       <div
         aria-labelledby={headingId}
-        className={classNames(classes.sectionContent)}
+        className={classNames(
+          classes.sectionContent,
+          styles.sectionContent,
+          onMounted({
+            [styles.sectionContentCollapsed]: !open,
```

Comment:
If we change the `sectionContent` as suggested above, then this should be changed to `sectionContentExpanded` (and adjusted accordingly)

Reply:
I've done it slightly differently. I've kept this line and removed `.sectionContent` completely - and now I'm going to hope you don't tell me that will break lazy loading. I tried testing it locally and it seemed fine.

Reply:
Looks fine, but run the tests and find out :wink:

Reply:
I updated the snapshots so I'm hoping that's all good

Reply:
I meant UI tests

Reply:
I ran all the public tests and create_data_block_with_chart and everything passed. But create_data_block_with_chart was previously passing locally before - the failure was only happening on the pipeline.

Reply:
Cool, all good then :+1: